config/foreman-proxy-content.migrations/230207194222-add-ssh-keepalive-ansible.rb

```diff
@@ -0,0 +1,5 @@
+ansible_module = 'foreman_proxy::plugin::ansible'
```

Comment:
You could even further optimize it and take the actual answer. The performance benefit is tiny, but it's more about optimizing the readability.
```suggestion
ansible_module = answers['foreman_proxy::plugin::ansible']
```